routes/variableurl.go

```diff
@@ -0,0 +1,123 @@
+package routes
+
+import (
+	"net/http"
+	"strconv"
+	"strings"
+)
+
+type (
+	parseIndex uint
```

Comment:
If you want to create an atom/symbol, you should rather use `struct{}`, like so.
```go
type parseIndex struct{}
```
And then use the `parseIndex{}` value.